core/server/api/v2/utils/serializers/input/integrations.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash');
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:input:integrations');
+
+module.exports = {
+    add(apiConfig, frame) {
+        debug('add');
+
+        frame.data = _.pick(frame.data.integrations[0], apiConfig.data);
```

Comment:
Was there a reason to not put the picking logic to shared?

Reply:
Gonna raise this in a separate PR :+1: